django_extensions/compat.py

```diff
@@ -145,19 +145,19 @@ def get_model_compat(app_label, model_name):
         return get_model(app_label, model_name)
 
 
-def get_models_compat(app_label):
-    """Get models on multiple Django versions."""
+def get_models_compat(app):
+    """Get models on multiple Django versions for an app."""
     try:
         # django >= 1.7
         from django.apps import apps
-        return apps.get_app_config(app_label).get_models()
+        return apps.get_app_config(app).get_models()
```

Comment:
https://github.com/django/django/blob/master/django/apps/registry.py#L140

Django calls it app_label here as well and it seems that that function won't work with anything but a string (aka app_label ?).